src/main/scala/com/gu/mobile/content/notifications/ContentLambda.scala

```diff
@@ -7,8 +7,13 @@ object ContentLambda extends Lambda {
 
   def processContent(content: Content): Boolean = {
     logger.info(s"Processing ContendId: ${content.id} Published at: ${content.getLoggablePublicationDate}")
+
+    // Temporary: this is to support the mobile text-to-speech experiment
+    sqs.sendMessage(content.id)
```

Comment:
Wrap this to catch/suppress any errors?

Reply:
I think that's a good point - I wondered if we might get messages with the same content id if an article was updated (after first being published). In this case we may want to trigger the audio-generator lambda to make sure the audio includes the latest content. The other kind of thing we were trying to do here is make any temporary extension to this lambda as small as possible, then handle any ugliness/complexity in the audio-generator lambda. 

As part of the of the experiment we also made a note that we might like to send messages (e.g. to our google space) when an audio conversion task was kicked off, so maybe we'd be able to see if this kind of behaviour (multiple subsequent content id appearing on the queue in rapid succession) happens when we've deployed this to the wild.